projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -6,7 +6,9 @@
         [value]="formatter ? formatter(value) : column.dataType === 'number' ? (value | igxdecimal: grid.locale) : column.dataType === 'date' ? (value | igxdate: grid.locale) : value"
         [row]="rowData" [column]="this.column.field" [containerClass]="'igx-grid__td-text'" [metadata]="searchMetadata"
         class="igx-grid__td-text">{{ formatter ? formatter(value) : column.dataType === 'number' ? (value | igxdecimal:
-        grid.locale) : column.dataType === 'date' ? (value | igxdate: grid.locale) : value }}</div>
+        grid.locale) : column.dataType === 'date' ? (value | igxdate: grid.locale) : column.dataType === 'boolean' ? "" : value }}
+        <igx-icon *ngIf="column.dataType === 'boolean'" [ngClass]="{'igx-grid__icon--checked': value}" >{{value ? 'check' : 'close'}}</igx-icon>
```

Comment:
This icon should not be rendered inside the wrapping div. It should be rendered outside of it.